website/content/api-docs/secret/databases/postgresql.mdx

```diff
@@ -55,6 +55,11 @@ has a number of parameters to further configure a connection.
   and password fields. See the [databases secrets engine docs](/vault/docs/secrets/databases#disable-character-escaping)
   for more information. Defaults to `false`.
 
+- `password_encryption` `(string: "plaintext")` - When set to "scram-sha-256", passwords will be encrypted by Vault and stored as-is by PostgreSQL.
```

Comment:
I'd suggest changing this to, for example, `password_authentication`, and having the values match https://www.postgresql.org/docs/current/auth-password.html -- so, plaintext would be `password`.

Reply:
`password_encryption` comes from [here](https://www.postgresql.org/docs/current/runtime-config-connection.html#GUC-PASSWORD-ENCRYPTION) and I think this is a _more_ accurate name than `password_authentication`. As far as the name of the value, I'm open to to changing it. It's currently `none` as @maxcoulombe suggested.

Reply:
Would you prefer something like `password_hash` to `password_authentication`?

Reply:
I like `password_authentication` more than `password_hash`, but I like either more than `password_encryption`.

Reply:
Sounds good to me!

Reply:
updated!